assets/kube-apiserver/kube-apiserver-deployment.yaml

```diff
@@ -122,6 +123,13 @@ spec:
             fi
             sleep 1
           done
+          while true; do
+            if oc replace --subresource=status -f ./99_feature-gate.yaml; then
+            echo "FeatureGate status applied successfully."
```

Comment:
I'm not sure if you need to indeed this line.  Probably best to do so to avoid any formatting problems.

Reply:
Good catch.